code/obj/item/uplinks.dm

```diff
@@ -815,15 +815,16 @@ Note: Add new traitor items to syndicate_buylist.dm, not here.
 						var/mob/living/carbon/human/H = M
 						var/obj/item/parts/HP = delivery
 					//	var/limb_name = HP.holder.real_name + "'s " + HP.name
-						if(HP == B.item) //Uhh idk if this will work
+						if(HP == B.item && HP.holder == M) //Is this the right limb and is it attached?
 							HP.remove()
 							take_bleeding_damage(H, null, 10)
 							H.changeStatus("weakened", 3 SECONDS)
 							playsound(H.loc, 'sound/impact_sounds/Flesh_Break_2.ogg', 50, 1)
 							H.emote("scream")
 							logTheThing("combat", user, null, "spy thief claimed [constructTarget(H)]'s [HP] at [log_loc(user)]")
-						else
+						else if(HP != B.item)
 							user.show_text("That isn't the right limb!", "red")
```

Comment:
no it's the left limb

Reply:
![image](https://user-images.githubusercontent.com/3855802/182046491-014531c0-2c54-4386-b41c-478203e622b0.png)